Extension/src/githubAPI.ts

```diff
@@ -274,13 +276,13 @@ async function getReleaseJson(): Promise<Build[]> {
     try {
         releaseJson = JSON.parse(data);
     } catch (error) {
-        throw new Error('Failed to parse release JSON');
+        throw new Error(localize("failed.parse.release", 'Failed to parse release JSON'));
     }
 
     // Type check
     if (isArrayOfBuilds(releaseJson)) {
         return releaseJson;
     } else {
-        throw new Error('Release JSON is not of type Build[]');
+        throw new Error(localize('failed.release.type', 'Release JSON is not of type {0}', "Build[]"));
```

Comment:
These errors get logged in telemetry -- I think we need to translate them later on after they haven't been logged to telemetry.

Reply:
I don't believe any of these errors in this file are user-facing.  They're just for telemetry.